packages/tom-server/src/types.ts

```
@@ -1,318 +1,294 @@
-import type { PathOrFileDescriptor } from 'fs'
-import type { SendMailOptions } from 'nodemailer'
-import { UserProfileSettingsPayloadT } from './user-info-api/types'
-
-export type expressAppHandler = _expressAppHandler
-export type AuthenticationFunction = MUtils.AuthenticationFunction
+import type { IdentityServerDb, Config as MConfig, Utils as MUtils } from "@twake/matrix-identity-server";
+import { type expressAppHandler as _expressAppHandler, errCodes } from "@twake/utils";
+import type { NextFunction, Request, Response } from "express";
+import type { PathOrFileDescriptor } from "fs";
+import type { SendMailOptions } from "nodemailer";
+import { UserProfileSettingsPayloadT } from "./user-info-api/types";
```

Biome (2.4.9)

[error] This import is unused.

(lint/correctness/noUnusedImports)